content/200-concepts/100-components/01-prisma-schema/10-postgresql-extensions.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: 'PostgreSQL extensions'
+metaDescription: 'How to represent PostgreSQL extensions in your Prisma schema'
+hidePage: false
+preview: true
+tocDepth: 2
+---
+
+<TopBlock>
+
+This page introduces PostgreSQL extensions and describes how to represent extensions in your Prisma schema, how to introspect existing extensions in your database, and how to apply changes to your extensions to your database with Prisma Migrate.
```

Comment:
Let's add a link to the Postgres docs on extensions, in order to make it clear up front that we are talking about official, Postgres-created extensions.